centreon/www/include/configuration/configObject/host/listHost.php

```diff
@@ -423,9 +423,9 @@
             'RowMenu_icone' => $host_icone,
             'RowMenu_link' => 'main.php?p=' . $p . '&o=c&host_id=' . $host['host_id'],
             'RowMenu_desc' => $host['host_alias'],
-            'RowMenu_address' => CentreonUtils::escapeSecure($host['host_address']),
+            'RowMenu_address' => $host['host_address'],
             'RowMenu_poller' => $tab_relation[$host['host_id']] ?? '',
-            'RowMenu_parent' => CentreonUtils::escapeSecure($tplStr),
+            'RowMenu_parent' => $tplStr,
```

Comment:
This will gonna break the link to the template
(I'm doing a ticket for this issue on develop)

<img width="1398" height="67" alt="image" src="https://github.com/user-attachments/assets/1cc86f7a-de9c-41d5-94af-dc5c17963edb" />